### PR TITLE
Adding sandbox checkbox and modifying widgets URL based off checkbox value

### PR DESCRIPTION
**Pull request by mishaber**
*Description of changes:*
Currently, the plugin only supported live pay environments, testing of the sandbox environment was not supported unless you manually changed the URL for widgets.js. The URL for loading the widgets javascript code was hard coded to load the production JS. With the addition of the sandbox checkbox, we can load the sandbox javascript URL when needed. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: shortcodes/express-pay.php [hunk 1]
```diff
@@ -14,6 +14,7 @@ function express_pay_handler($args) {
 	$lwaClientId =  get_option('amzn_lwa_client_id');
 	$accessKey =  get_option('amzn_access_key'); 
 	$secretKey =  get_option('amzn_secret_key');
+	$sandbox =  get_option('amzn_sandbox');
 	$note = ($args["note"] != '') ? $args["note"] : "purchase on " . get_home_url(); 
 	$amount = ($args["amount"] != '') ? $args["amount"] : "1";
```
Comment (islandskater43) [inline shortcodes/express-pay.php]: Do we need to introduce anything within the parameters to support the switch to sandbox?

Comment (islandskater43) [reply shortcodes/express-pay.php]: Parameters referenced here are: https://github.com/amzn/pay-with-amazon-express-wordpress/blob/c230af83d70aaa0700d7071ff8d775405c9e8302/shortcodes/express-pay.php#L21-L29

**Comment by islandskater43** [comment]
Can we fix the spacing so that 35 & 36 line up with the rest?
https://github.com/amzn/pay-with-amazon-express-wordpress/blob/c230af83d70aaa0700d7071ff8d775405c9e8302/shortcodes/express-pay.php#L34-L37

File: shortcodes/express-pay.php [hunk 2]
```diff
@@ -31,8 +32,9 @@ function express_pay_handler($args) {
...
 	$signature = _urlencode(_signParameters($parameters, $secretKey));
 	$parameters['signature'] = $signature;
-  $buttonId = uniqid();
-	$button = "<script type='text/javascript' src='https://static-na.payments-amazon.com/OffAmazonPayments/us/js/Widgets.js'></script>
+    $buttonId = uniqid();
+    $widgetsUrl = $sandbox ? 'https://static-na.payments-amazon.com/OffAmazonPayments/us/sandbox/js/Widgets.js' : 'https://static-na.payments-amazon.com/OffAmazonPayments/us/js/Widgets.js';
+	$button = "<script type='text/javascript' src='{$widgetsUrl}'></script>
 		<div id='AmazonPayButton" . $buttonId . "'></div>
 		<script type='text/javascript'>
 			OffAmazonPayments.Button('AmazonPayButton" . $buttonId . "', 'A1GV76EFH0T2Y7', {
```
[the remark this answers is not in the document]
Comment (mishaber) [reply shortcodes/express-pay.php]: Updated the code to use the $sellerId value

File: shortcodes/express-pay.php (hunk 1 above)
Comment (mishaber) [reply shortcodes/express-pay.php]: If I try to reference a new parameter for sandbox, I get this error: InvalidParameter Invalid parameters passed, please only provide parameters specified in the integration guide 

It doesn't look like there is anything for environment supported: https://pay.amazon.com/us/developer/documentation/express/201737790

I will however remove the platformId parameter that is commented out.

**Pull request merged**